src/styles/styles.js

```diff
@@ -1580,6 +1580,33 @@ const styles = {
         height: 80,
     },
 
+    chatAvatar: {
+        height: 52,
+        width: 52,
+        borderRadius: 100,
+        borderColor: colors.white,
+        borderWidth: 3,
+        marginLeft: -15,
```

Comment:
I assume this is to make sure there is overlap when there is multiple avatars? What happens when there is just one avatar? Also, I think this should be -16: in Figma, you will see that the avatars are -8 apart that doesn't include 4px worth of border from each avatar.